.github/workflows/twister_tests_blackbox.yml

```diff
@@ -43,6 +43,7 @@ jobs:
         echo "$HOME/.local/bin" >> $GITHUB_PATH
 
         west init -l . || true
+        west config manifest.group-filter -- +ci,-optional
```

Comment:
`-optional` is redundant right? that should be the default

not asking to change it just trying to understand

Reply:
> `-optional` is redundant right? that should be the default

is it the default? defined where? AFAIK, without filters, everything is fetched by default. see `.github/workflows/compliance.yml` for example.

Either way, this fix is not enough, we have a "bigger" problem....

Reply:
> is it the default? defined where?

Here? https://github.com/zephyrproject-rtos/zephyr/blob/main/west.yml#L27 if that's not effective you want to filter out babblesim as well

Reply:
oh yeah, you are right. Then I have to find a more directy approach now :(